### DISCUSSION TITLE
How to click element when going through the tour.

### DISCUSSION THREAD

**Discussion by vishnup-ditto** [elrumordelaluz/reactour#520] [Q&A]
Suppose I am going through the tour and reach a place where I can show some expanded content via a button click, imagine filters

![image](https://user-images.githubusercontent.com/113330036/206168489-2d4ebda3-293f-4671-b7b8-4ab67c6101d5.png)

How do I click the button, to show the filters and guide them through it? I tried the `action` prop in steps but that doesn't have access to `.click()`. Any help please.

**Comment by elrumordelaluz** [comment]
Hi @vishnup-ditto. Mind creating a minimal reproduction in a sandbox in order to allow others to debug your use-case and try to find a solution? Thanks!

**Reply by vishnup-ditto** [reply]
Hey, you can close this discussion, no longer using the library.
